src/spikeinterface/extractors/phykilosortextractors.py

```diff
@@ -178,6 +179,13 @@ def __init__(
 
         self.add_sorting_segment(PhySortingSegment(spike_times_clean, spike_clusters_clean))
 
+        # Caching spike vector for faster computation.
```

Comment:
You can use the `self.to_spike_vector` method with cache=True to avoid duplication.

Reply:
If I call `to_spike_vector`, it's not going to compute it efficiently.
Here I'm using the data that's already loaded in the files!

Reply:
Makes sense. Can you add a comment along those lines so nobody stumbles like myself in the future. Like:

# Using properties available at __init__ to build the a cached spike_vector and avoid computation.